gammagl/layers/conv/hgt_conv.py

```diff
@@ -0,0 +1,151 @@
+import math
+import tensorlayerx as tlx
+from gammagl.layers.conv import MessagePassing
+from tensorlayerx.nn import ModuleDict, Linear, Parameter, ParameterDict
+from gammagl.utils import segment_softmax
+
+
+class HGTConv(MessagePassing):
+    r"""The Heterogeneous Graph Transformer (HGT) operator from the
+    `"Heterogeneous Graph Transformer" <https://arxiv.org/abs/2003.01332>`_
+    paper.
+
+    Args:
+        in_channels (int or Dict[str, int]): Size of each input sample of every
+            node type, or :obj:`-1` to derive the size from the first input(s)
+            to the forward method.
+        out_channels (int): Size of each output sample.
+        metadata (Tuple[List[str], List[Tuple[str, str, str]]]): The metadata
+            of the heterogeneous graph, *i.e.* its node and edge types given
+            by a list of strings and a list of string triplets, respectively.
+            See :meth:`torch_geometric.data.HeteroData.metadata` for more
```

Comment:
Replace "torch_geometric.data.HeteroData.metadata" with "gammagl.XXX". What is more, I think word "meth" is an error.